dbschema/1.tables.sql

```diff
@@ -20,3 +20,22 @@ CREATE INDEX tbl_token_transfers_value ON tbl_token_transfers (value);
 CREATE INDEX tbl_token_transfers_block_date ON tbl_token_transfers (block_date);
 CREATE INDEX tbl_token_transfers_block_number ON tbl_token_transfers (block_number);
 CREATE INDEX tbl_token_transfers_block_hash ON tbl_token_transfers (block_hash);
+
+CREATE TABLE tbl_token_metadata (
```

Comment:
table name should be plural. I would actually just call this `tbl_tokens` I think

Reply:
`tbl_token_metadatas`